addons/account/models/account.py

```diff
@@ -714,16 +715,25 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
             if tax.price_include:
                 total_excluded -= tax_amount
                 base -= tax_amount
+                tax_base -= tax_amount
+                tax_excluded -= tax_amount
             else:
                 total_included += tax_amount
+                tax_included += tax_amount
 
             if tax.include_base_amount:
                 base += tax_amount
 
+            if tax.amount_type in ['division']:
+                tax_base += tax_amount
+
             taxes.append({
                 'id': tax.id,
                 'name': tax.with_context(**{'lang': partner.lang} if partner else {}).name,
                 'amount': tax_amount,
+                'base': tax_base,
+                'tax_excluded': currency.round(tax_excluded) if round_total else tax_excluded,
```

Comment:
As mentioned above I am not yet sure to keep it there but it represents the computed value on an isolated tax level (base is the base to calculate, tax_excluded and tax_included are the representation in these terms)